conf/messages

```diff
@@ -479,5 +479,48 @@ timeoutDialog.sign-out-button-text.logged-out = Delete your answers
 delete-answers.heading = For your security, we signed you out
 delete-answers.message = We did not save your answers.
 
-
-
+# dates
+date.january=January
+date.february=February
+date.march=March
+date.april=April
+date.may=May
+date.june=June
+date.july=July
+date.august=August
+date.september=September
+date.october=October
+date.november=November
+date.december=December
+
+date.ordinal.1=st
+date.ordinal.2=nd
+date.ordinal.3=rd
+date.ordinal.4=th
+date.ordinal.5=th
+date.ordinal.6=th
+date.ordinal.7=th
+date.ordinal.8=th
+date.ordinal.9=th
+date.ordinal.10=th
+date.ordinal.11=th
+date.ordinal.12=th
+date.ordinal.13=th
+date.ordinal.14=th
+date.ordinal.15=th
+date.ordinal.16=th
+date.ordinal.17=th
+date.ordinal.18=th
+date.ordinal.19=th
+date.ordinal.20=th
+date.ordinal.21=st
+date.ordinal.22=nd
+date.ordinal.23=rd
+date.ordinal.24=th
+date.ordinal.25=th
+date.ordinal.26=th
+date.ordinal.27=th
+date.ordinal.28=th
+date.ordinal.29=th
+date.ordinal.30=th
+date.ordinal.31=st
```

Comment:
Minor concern this is technically repeating content. potentially use a match statement in the code to decide which original to append ?

Reply:
Just realised this may not be possible due to the welsh translation not having the same pattern as English.

Reply:
Yeah, was wondering that myself. The language rules are strange and arbitrary though, so the code would be pretty hard to follow and understand, but spelling it out like this in the messages files makes it easy to follow at a glance.

This approach was also taken in the gforms project: https://github.com/hmrc/gform-frontend/blob/11ebc6aab14bf064c5332790fc135cc3fed87e39/conf/messages#L214-L259